### DISCUSSION TITLE
In Auto3dSeg's "Algorithm Generation" part, why doesn't SwinUNETR use the transforms like the other algorithms do?

### DISCUSSION THREAD

**Discussion by abcvav** [Project-MONAI/tutorials#1012] [Q&A] [answered]
Hi everyone,

In Auto3dSeg, on [this page](https://github.com/Project-MONAI/tutorials/blob/main/auto3dseg/docs/algorithm_generation.md), DiNTS, 2D SegResNet and SegResNet all apply these transforms:
- Random Gaussian smoothing
- Random intensity scaling
- Random Gaussian noising

While SwinUNETR doesn't apply these transforms, I am confused and wonder why SwinUNETR does not need them. Does anybody have a clue?

![image](https://user-images.githubusercontent.com/47599849/197454675-bb1c094b-f9f7-4f3d-8ca9-cc2664f81b74.png)

Thanks a lot for any help!

Best

**Comment by tangy5** [comment, marked as answer]
Hi @abcvav , thanks for the question, the four algorithms are optimized based on experimental practice, the transform will be a little bit different. The SwinUNETR contains augmentations including rotation, flipping, intensity shifting based on the tutorial and the publications. We tried adding more augmentation combinations, the results do not change much. Thanks.

**Reply by abcvav** [reply]
Thank you for answering! I got it now
